atd-vze/src/views/Crashes/Crash.js

```diff
@@ -29,17 +30,36 @@ const calculateYearsLifeLost = people => {
 
 function Crash(props) {
   const crashId = props.match.params.id;
-  const { loading, error, data } = useQuery(GET_CRASH, {
+  const { loading, error, data, refetch } = useQuery(GET_CRASH, {
     variables: { crashId },
   });
 
   if (loading) return "Loading...";
   if (error) return `Error! ${error.message}`;
 
+  const createGeocoderAddressString = data => {
+    const geocoderAddressFields = [
```

Comment:
Just a heads up that this might need updating once we start referencing a pre-concatenated address field from the DB.

Reply:
👍 and this also may need to be removed if it is confusing to the user.